### PR TITLE
- fixed out of memory exception crash while html report generation in…

### PR DESCRIPTION
**Pull request by Yaroshvitaliy**
… case it's too huge

- changed json serialization way for html report

### CODE REVIEW

File: src/NBomber/DomainServices/Reporting/CsvReport.fs [hunk 1]
```diff
@@ -23,7 +25,7 @@ let private toMb (bytes: int64) =
     bytes |> Statistics.Converter.fromBytesToMb
 
 let private getLine (scenarioName: string, duration: TimeSpan, stats: StepStats, testInfo: TestInfo) =
-    let format = seq {0 .. 20} |> Seq.map(sprintf "{%i}") |> String.concat(separator) // {0},{1},{2},...
+    let format = seq {0 .. 20} |> Seq.map(fun x -> x.ToString()) |> String.concat(separator) // {0},{1},{2},...
```
Comment (AntyaDev) [inline src/NBomber/DomainServices/Reporting/CsvReport.fs:28]: can it be just list?
[0 .. 20]

File: tests/NBomber.IntegrationTests/Reporting/ReportingConfigurationTests.fs [hunk 2]
```diff
@@ -91,3 +94,77 @@ let ``withReportFileName and withReportFolder should be properly handled`` () =
...
+    test <@ csvReportPrinted = false @>
+    test <@ htmlReportPrinted = false @>
+    test <@ mdReportPrinted = false @>
+    test <@ consoleReportPrinted = false @>
+
+[<Fact>]
+let ``withoutReports should not save report files`` () =
+
+    // delete all directories with all files
+    if Directory.Exists "./no-reports" then
```
Comment (AntyaDev) [inline tests/NBomber.IntegrationTests/Reporting/ReportingConfigurationTests.fs:143]: please add some prefix for folders names like here
https://github.com/PragmaticFlow/NBomber/blob/dev/tests/NBomber.IntegrationTests/StepTests.fs#L334

it prevents concurrent access to the same folder

**Review by AntyaDev — CHANGES_REQUESTED** [review]

File: src/NBomber/DomainServices/Reporting/CsvReport.fs (hunk 1 above)
Comment (Yaroshvitaliy) [reply src/NBomber/DomainServices/Reporting/CsvReport.fs]: fixed

File: tests/NBomber.IntegrationTests/Reporting/ReportingConfigurationTests.fs (hunk 2 above)
Comment (Yaroshvitaliy) [reply tests/NBomber.IntegrationTests/Reporting/ReportingConfigurationTests.fs]: there is only one test with such folder name, but anyway, fixed

**Pull request merged**